### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by sunga12**
This project has a Postgres database set up.
Project schema built:

- Created and ran the necessary migration files.
- Table and column names match the ERD diagram.
- The photo column for users table will be a link to an image
- Foreign keys should be included.
- In the posts table the column holding the foreign key to the users table is named author_id
- All columns that are foreign keys have a corresponding index.

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @sunga12,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that does not have a descriptive README.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/migrate/20230920100700_add_details_to_comments.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class AddDetailsToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :user_id, :bigint
+    add_index :comments, :user_id
+    add_column :comments, :post_id, :bigint
+    add_index :comments, :post_id
+    add_column :comments, :text, :text
+  end
```
Comment (TadesseDev) [inline db/migrate/20230920100700_add_details_to_comments.rb:8]: - [Optional] Dear @sunga12, I see you created a table without a column and applied a `change` method to add the columns. Usually `change` method is used to add a column that is missed or some other changes to the table that should be applied. Please consider creating a table with all its columns.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,211 @@
...
+
+I would like to thank microverse for this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (TadesseDev) [inline README.md:209]: - [ ] Dear @sunga12, Great you have added a readme file, But currently your MIT linsense file link is broken. For a professional readme documentation, we need to have a working link all the way. Please address this.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @sunga12 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x]  Most of the requirement has been meet. ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230920100700_add_details_to_comments.rb (hunk 1 above)
Comment (sunga12) [reply db/migrate/20230920100700_add_details_to_comments.rb:8]: Noted thanks. will do going forward

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @sunga12 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested change. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**